src/ganj/HumanRiddleMaker.java

```diff
@@ -0,0 +1,30 @@
+package ganj;
+
+import java.util.Scanner;
+
+public class HumanRiddleMaker implements RiddleMaker {
+    private int secretNumber;
+    @Override
+    public void thinkOfNumberBetween(int from, int to) {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.println("Let's guess a number from " + from + " to " + to);
```

Comment:
I propose here just print something like `Guess the number from ... to ... and press enter when you are ready`. And just wait for enter.